src/components/Calculator.js

```diff
@@ -1,60 +1,51 @@
-          <button type="submit" onClick={this.handleClick} className="num">9</button>
-          <button type="submit" onClick={this.handleClick} className="num operator">x</button>
-          <button type="submit" onClick={this.handleClick} className="num">4</button>
-          <button type="submit" onClick={this.handleClick} className="num">5</button>
-          <button type="submit" onClick={this.handleClick} className="num">6</button>
-          <button type="submit" onClick={this.handleClick} className="num operator">-</button>
-          <button type="submit" onClick={this.handleClick} className="num">1</button>
-          <button type="submit" onClick={this.handleClick} className="num">2</button>
-          <button type="submit" onClick={this.handleClick} className="num">3</button>
-          <button type="submit" onClick={this.handleClick} className="num operator">+</button>
-          <button type="submit" onClick={this.handleClick} className="zero">0</button>
-          <button type="submit" onClick={this.handleClick} className="num">.</button>
-          <button type="submit" onClick={this.handleClick} className="num operator">=</button>
+  };
+  return (
+    <>
+      <div className="calc-container">
+        <div className="calc-input">
+          {state.total}
+          {state.operation}
+          {state.next}
         </div>
-      </>
-    );
-  }
-}
+        <button type="submit" onClick={handleClick} className="num top-operator">AC</button>
+        <button type="submit" onClick={handleClick} className="num top-operator">+/-</button>
+        <button type="submit" onClick={handleClick} className="num top-operator">%</button>
+        <button type="submit" onClick={handleClick} className="num operator">÷</button>
+        <button type="submit" onClick={handleClick} className="num">7</button>
+        <button type="submit" onClick={handleClick} className="num">8</button>
+        <button type="submit" onClick={handleClick} className="num">9</button>
+        <button type="submit" onClick={handleClick} className="num operator">x</button>
+        <button type="submit" onClick={handleClick} className="num">4</button>
+        <button type="submit" onClick={handleClick} className="num">5</button>
+        <button type="submit" onClick={handleClick} className="num">6</button>
+        <button type="submit" onClick={handleClick} className="num operator">-</button>
+        <button type="submit" onClick={handleClick} className="num">1</button>
+        <button type="submit" onClick={handleClick} className="num">2</button>
+        <button type="submit" onClick={handleClick} className="num">3</button>
+        <button type="submit" onClick={handleClick} className="num operator">+</button>
+        <button type="submit" onClick={handleClick} className="zero">0</button>
+        <button type="submit" onClick={handleClick} className="num">.</button>
+        <button type="submit" onClick={handleClick} className="num operator">=</button>
```

Comment:
- [ ] (Optional) You can consider putting all the mathematical symbols/numbers in an array and then iterate through them using the `map` function. You can read this [article](https://www.pluralsight.com/guides/how-to-use-the-map()-function-to-export-javascript-in-react) to get a better understanding 👍.